cmd/conf/chain.go

```diff
@@ -4,32 +4,39 @@
 package conf
 
 import (
+	"time"
+
 	"github.com/offchainlabs/nitro/arbnode"
 	"github.com/offchainlabs/nitro/cmd/genericconf"
+	"github.com/offchainlabs/nitro/util/rpcclient"
 	flag "github.com/spf13/pflag"
 )
 
 type L1Config struct {
-	ChainID            uint64                        `koanf:"chain-id"`
-	Rollup             arbnode.RollupAddressesConfig `koanf:"rollup"`
-	URL                string                        `koanf:"url"`
-	ConnectionAttempts int                           `koanf:"connection-attempts"`
-	Wallet             genericconf.WalletConfig      `koanf:"wallet"`
+	ChainID    uint64                        `koanf:"chain-id"`
+	Rollup     arbnode.RollupAddressesConfig `koanf:"rollup"`
+	Connection rpcclient.ClientConfig        `koanf:"connection" reload:"hot"`
+	Wallet     genericconf.WalletConfig      `koanf:"wallet"`
+}
+
+var L1ConnectionConfigDefault = rpcclient.ClientConfig{
+	URL:            "",
+	Retries:        2,
```

Comment:
We might want to keep the existing default of 15 -- the L1 node can take a while to come up, or it can take a while for the networking stack to come up, etc.

Reply:
retries is for an rpc request sent on an active connection that times out (like the HarryNg bug)

ConnectionWait is how long the other side has to succeed connection. By default set to 1 minute.